imitation/experiments.py

```diff
@@ -0,0 +1,217 @@
+import sacred
+
+# NOTE: To call `train_exp.run()`, must import `imitation.scripts.train`.
+# A quick way to do this is to `from imitation.scripts.train import train_exp`.
+train_exp = sacred.Experiment("train", interactive=True)
+
+# NOTE: To call `data_collection.run()`, must import
+# `imitation.scripts.data_collection`.
+data_collection_exp = sacred.Experiment("data_collection")
+
+
+DEFAULT_BLANK_POLICY_KWARGS = dict(
+    learning_rate=3e-4,
+    nminibatches=32,
+    noptepochs=10,
+    # WARNING: this is actually 8*256=2048 due to 8 vector environments
+    n_steps=256,
+)
+
+
+@train_exp.config
+def train_defaults():
+    n_epochs = 50
```

Comment:
I'll leave comments for later, since I'm expecting some of these parameters to change as we transition to training from demonstrations (as opposed to policies).